scripts/remove

```diff
@@ -41,10 +41,10 @@ ynh_remove_nginx_config
 ynh_script_progression --message="Removing PHP-FPM configuration..." --weight=2
 
 # Remove the app dependencies installed along with php-fpm
-ynh_remove_app_dependencies
+# ynh_remove_app_dependencies
```

Comment:
Please see : https://github.com/YunoHost-Apps/garradin_ynh/pull/62

This line was added to fix something

Reply:
I commented this line because with the remove script I have seen this message:
`WARNING E: Unable to locate package garradin-ynh-deps` 
and like the install script I have this line:
`ynh_remove_fpm_config --package="$extra_php_dependencies"`